api/cluster.go

```diff
@@ -563,6 +563,12 @@ func queryPeers(ctx context.Context, peerGroups map[int32][]cluster.Node, name s
 		}
 
 		for group, peers := range peerGroups {
+			if len(peers) == 0 {
+				log.Warningf("Peer group %d has no peers", group)
```

Comment:
logs should always have a prefix
```
api/cluster.go
378:		log.Errorf("HTTP getData() %s", err.Error())
426:	log.Debugf("HTTP %s across %d instances", name, len(peers)-1)
443:			log.Debugf("HTTP Render querying %s%s", peer.GetName(), path)
447:				log.Errorf("HTTP Render error querying %s%s: %q", peer.GetName(), path, err.Error())
509:		log.Errorf("HTTP peerQuery unable to get peers, %s", err.Error())
551:			//log.Debugf("HTTP Render querying %s%s", peer.GetName(), path)
560:				log.Errorf("Peer %s responded with error = %q", peer.GetName(), err.Error())
567:				log.Warningf("Peer group %d has no peers", group)
659:				log.Warnf("Something weird: %v", originalPeers)

```

Reply:
sorry, i'm not completely sure how to understand your message. usually our prefix is just `functionName:` right? do you mean to say that I should update all of those log statements in this PR?

Reply:
just put "HTTP " as prefix in your log messages. don't worry about consistency across the entire codebase